### PR TITLE
added data-format restructuring

### PR DESCRIPTION
**Pull request by aaishikdutta**
With the addition of multiple loaders and chunkers the embedchain.py size seems to be growing and I feel it would add to the code-redability by adding some sort of abstraction to the application.

### CODE REVIEW

**Review by taranjeet — APPROVED** [review]
PR looks good to me and makes much more sense.
The only comment that I have is regarding the name. 
whether it should be "data_format" or "data_formatter"

@cachho what do you think?

File: embedchain/data_format/data_format.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    """
+    DataFormat is an internal utility class which abstracts the mapping for
+    loaders and chunkers to the data_type entered by the user in their
+    .add or .add_local method call 
+    """
+    def __init__(self, data_type):
+        self.loader = self._get_loader(data_type)
+        self.chunker = self._get_chunker(data_type)
+        
+    def _get_loader(self, data_type):
```
Comment (cachho) [inline embedchain/data_format/data_format.py:25]: pretty sure you can make this a static method

Comment (cachho) [reply embedchain/data_format/data_format.py:25]: #179

**Comment by cachho** [comment]
we don't have to do my proposed changes in this PR. Looks good to me. About the name, classes should be in PascalCase. DataFormatter makes sense since it's Chunker and Loader as well.

**Review by taranjeet — APPROVED** [review]

**Pull request merged**